go/test/endtoend/vreplication/wrappers_test.go

```diff
@@ -395,20 +402,22 @@ func (v VtctldReshard) Create() {
 	if v.skipSchemaCopy {
 		args = append(args, "--skip-schema-copy="+strconv.FormatBool(v.skipSchemaCopy))
 	}
+	args = append(args, v.createFlags...)
 	v.exec(args...)
 }
 
 func (v VtctldReshard) SwitchReadsAndWrites() {
-	v.exec("SwitchTraffic")
+	args := []string{"SwitchTraffic"}
+	args = append(args, v.switchFlags...)
+	v.exec(args...)
 }
 
 func (v VtctldReshard) ReverseReadsAndWrites() {
 	v.exec("ReverseTraffic")
 }
 
 func (v VtctldReshard) Show() {
-	//TODO implement me
-	panic("implement me")
+	v.exec("Show")
```

Comment:
Does Show not take any other flags? Same question for ReverseTraffic above.

Reply:
It does take flags, FWIW:
```
❯ vtctldclient MoveTables show --help
Show the details for a MoveTables VReplication workflow.

Usage:
  vtctldclient MoveTables show

Aliases:
  show, Show

Examples:
vtctldclient --server localhost:15999 MoveTables --workflow commerce2customer --target-keyspace customer show

Flags:
  -h, --help             help for show
      --include-logs     Include recent logs for the workflow. (default true)
      --shards strings   (Optional) Specifies a comma-separated list of shards to operate on.
```

We might want to test `--shards` at some point here.